resources/js/components/RequestTabs.vue

```diff
@@ -1,13 +1,19 @@
 <script>
-import Dropdown from './../components/Dropdown';
-import FilesInput from './../components/FilesInput';
-import HeaderFields from './../components/HeaderFields';
+import Dropdown from './Dropdown';
+import HeaderFields from './HeaderFields';
+import BodyOptions from './request/BodyOptions';
+import BodyMultipartForm from './request/BodyMultipartForm';
+import BodyFormUrlEncoded from './request/BodyFormUrlEncoded';
+import BodyRawVue from './request/BodyRaw';
 
 export default {
     components: {
         'dropdown': Dropdown,
-        'files-input': FilesInput,
         'header-fields': HeaderFields,
+        'body-options': BodyOptions,
+        'body-multipart-form': BodyMultipartForm,
+        'body-from-url-encoded': BodyFormUrlEncoded,
```

Comment:
I guess the word `from` is a typo. let's change this to `body-form-url-encoded` ?

Reply:
Oops, sorry 😅
Fixed